docs/guide/tutorials.rst

```diff
@@ -9,134 +9,199 @@ Structure of Data Sets
+
+|	
 
 #. features
-	This folder contains features info in the 
-	`featureXML <https://raw.githubusercontent.com/OpenMS/OpenMS/develop/share/OpenMS/SCHEMAS/FeatureXML_1_9.xsd>`_ format.
+	This folder contains features info either in the 
+	`featureXML <https://raw.githubusercontent.com/OpenMS/OpenMS/develop/share/OpenMS/SCHEMAS/FeatureXML_1_9.xsd>`_ format or in 
+	`Chromeleon CDS7 TXT <https://www.thermofisher.com/order/catalog/product/CHROMELEON7>`_ format. 
+	Features contain the processed results of a single sample after applying a workflow and running the command ``STORE_FEATURES`` and
+	can be stored at both the sample (i.e., injection) or the sample group (merged injections) level.
+
+|
 
 #. mzML
-	This folder contains `mzML <https://www.psidev.info/mzML>`_ files which descripes raw spectrometer value, 
-	a single mzML file usually encapsulates all the
-	information extracted from a single MS run.
+	This folder contains mass spectrometry data files in the most widely open-source format : `mzML <https://www.psidev.info/mzML>`_ 
+	which descripes raw spectrometer value, a single mzML file usually encapsulates all the information extracted from a single MS run.
+
+	Converting raw files to the ``mzML`` format
+	~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	`Proteowizard <https://proteowizard.sourceforge.io/download.html>`_ is a toolkit to view and convert mass spectrometry
+	data. The tools kit includes SeeMS and MSConvert to visualize MS data and convert to and from ``mzML`` file format respectively.
+
+	Once installed, you will be presented with the following window:
+
+	.. image:: ../images/msconvert.png
+
+	To convert mass spectrometry files, please follow the steps below:
+
+	#. Select the file you wish to convert.
+	#. Add the selected file to the list of files to process.
+	#. Select the output directory where the converted files will be saved.
+	#. Select the output format, providing an extension to the file in not mandatory.
+	#. Check/uncheck compression and packaging options if desired.
+	#. Select how many files you wish to convert at once, then hit the "Start" button.
+
+
+|
 
 #. parameters.csv
-	This file contains a list of parameters for the RawDataProcessor functions, 
-	the values stored are organised by the header :
-	``function,name,type,value,default,restrictions,description,``
-	``used_,comment_,comparator``, a sample entry may look like this :
-	``MRMMapping,precursor_tolerance,float,``
-	``0.0009,0.1,0.1,Precursor tolerance when mapping (in Th),TRUE,,``.
+	This file contains a list of parameters for the workflow steps as well as various other application settings,
+	the header including a sample entry is shown below:
+
+	.. table:: parameters.csv Headers
+		:widths: auto
+
+		========== =================== ===== ====== ======= ============ ======================================== ===== ========= ==========
```

Comment:
Please remove the "comparator" column.  It is also a legacy header that is no longer used.  Also, it looks like "description" and "used_" are not separated properly.